docs/projects/ad77681evb/index.rst

```diff
@@ -0,0 +1,259 @@
+.. _ad77681evb:
+
+AD77681EVB HDL project
+=================================================================================
+
+Overview
+---------------------------------------------------------------------------------
+
+The :adi:`AD7768-1`/ :adi:`ADAQ7768-1` is a low power, high performance, Σ-Δ
+analog-to-digital converter (ADC), with a Σ-Δ modulator and digital filter for
+precision conversion of both ac and dc signals. The :adi:`AD7768-1` is a single
+channel version of the AD7768, an 8-channel, simultaneously sampling, Σ-Δ ADC.
+The :adi:`AD7768-1` provides a single configurable and reusable data acquisition
+footprint, which establishes a new industry standard in combined ac and dc
```

Comment:
```suggestion
footprint, which establishes a new industry standard in combined AC and DC
```

Reply:
Done.

Reply:
I don't know what happened between the squashes, but this change is still not applied

Reply:
Done.